### DISCUSSION TITLE
Extruder 100mm Very Slow

### DISCUSSION THREAD

**Discussion by JamieLee2k** [mriscoc/Ender3V2S1#1019] [General]
I have just updated to the latest firmware after installing a few new things like Micro Swiss Hot End and a Redrex All Metal extruder and so when I heat the nozzle and try to extrude 100mm it comes out slowly and also its more like 66mm, now normally I would just adjust it but on the last firmware (may) I had the same issue at first, when it came out slow it was always shorter and when I turned it off and on it would go fast then extrude close to 100mm, since I have a new extruder I will need to redo it and this is where my problem is.

Is this my fault because I am 100% sure I have installed everything correctly, I have watched multiple videos to double check and it seems I have done it.

**Comment by ifoundmikey** [comment]
Any solution? Having the same issue w/ my ender 3 v2 neo. Normally don’t do firmware updates, but had a situation where I had to, and now I’m up to 375mm to get close to 100mm filament eaten, but it takes forever, and only a thin string of filament leaves the nozzle. Changed nozzle, checked hot end, and when fed through by hand, it comes through like gangbusters, so no clog.  If I raise the flow to 125% it extrudes 25% longer, but still thin. Is there a setting I’m missing?

**Reply by mriscoc** [reply]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
